dev/public/index.html

```diff
@@ -15,8 +15,10 @@
     <nav class="bg-gray-800 pt-2 md:pt-1 pb-1 px-1 mt-0 h-auto fixed w-full z-20 top-0">
       <div class="flex flex-wrap items-center">
         <div class="flex flex-shrink md:w-1/3 justify-center md:justify-start text-white">
```

Comment:
We could remove the `md:w-1/3 class`, this will prevent to break the title on small screens

![image](https://user-images.githubusercontent.com/27200110/140657708-e21cc7ec-4d60-4ad3-891d-929fbc972132.png)

Reply:
Done